resolve.go

```diff
@@ -212,9 +206,9 @@ func handleNoWellKnown(serverName ServerName, host string) (results []Resolution
 			}
 
 			results = append(results, ResolutionResult{
-				Destination: fmt.Sprintf("%s:%d", target, rec.Port),
-				Host:        serverName,
-				Name:        string(serverName),
+				Destination:   net.JoinHostPort(target, strconv.Itoa(int(rec.Port))),
```

Comment:
given that we know that `target` is not an IP literal (since such things are invalid in SRV records), can we do this the easy way?

Reply:
Ah, right, good point.